app/src/main/java/chat/rocket/android/util/TokenSerialisableModel.kt

```diff
@@ -0,0 +1,36 @@
+package chat.rocket.android.util
+
+import java.io.Serializable
+
+class TokenSerialisableModel(
+    private val firstParam: String,
```

Comment:
We know that it is the fisrt parameter, the second, the fourth, and so on. Can you replace to something more meaningful?